pkg/schema/generate.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"github.com/alecthomas/jsonschema"
+	"io/ioutil"
+	"k8s.io/client-go/tools/clientcmd/api"
+	"sigs.k8s.io/yaml"
+	"strings"
+)
+
+const outpufFile = "site/content/usage/20-schema.md"
+
+func main() {
+
+	var document strings.Builder
+	document.WriteString("---\n")
+	document.WriteString("title: Config file schema\n")
+	document.WriteString("weight: 200\n")
+	document.WriteString("---\n\n")
```

Comment:
Minor nit: we could also use a template string and a single `WriteString` call for the content till here.

```golang
	b.WriteString(`---
title: Config file schema
weight: 200
---

`)
```

Reply:
Good idea, I forget about these, thanks